index.js

```diff
@@ -0,0 +1,122 @@
+
+//removes grid container from document flow
+document.getElementById('gridContainer').style.display="none"
+document.getElementById('gameTitle').style.display="none"
+
+
+//function to generate all game tiles
+document.querySelector('.form').addEventListener('submit', (e) => {
+    //stops form from submitting
+    e.preventDefault()
+    //Defining total grid size, number of lynx and misses from user input
+    let numLynx = parseInt(document.getElementById('numLynx').value)
+    let gridSize = parseInt(document.getElementById('gridSize').value)
+    let miss = gridSize - numLynx
+
+    //reset
+    document.getElementById('gridContainer').innerHTML = ''
+    document.getElementById('errorMessage').textContent = ''
+
+
+        let generateSquares = () => {
+            //create empty array
+            let squares = []
+            //For every hit we are putting a div into the array with data-hit 1
+            for (i = 0; i < numLynx; i++) {
+                squares.push('<div class="grid-item" data-hit="1"></div>')
+            }
+            //For every hit we are putting a div into the array with data-hit 0
+            for (i = 0; i < miss; i++) {
+                squares.push('<div class="grid-item" data-hit="0" ></div>')
+            }
+            //shuffle the array
+            let shuffle = (array) => array.map(a => ({
+                sort: Math.random(),
+                value: a
+            })).sort((a, b) => a.sort - b.sort).map(a => a.value)
+            let shuffledArr = shuffle(squares)
+            //adding each array element to html of container
+            shuffledArr.forEach((square) => {
+                document.getElementById('gridContainer').innerHTML += square
+            })
+        }
+
+    //validates form input
+    if(!validateInput(gridSize, numLynx)){
+        document.getElementById('errorMessage').textContent = errorMessage
+    }else {
+        //execute function
+        generateSquares()
+        turnCard()
```

Comment:
why is generate Squares a anonymous function inside the event listener but turnCard is a named function outside? seems inconsistent. same with the shuffle function.